examples/conventional-templates/subsrciption-benchmark.yaml

```
@@ -0,0 +1,91 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: subscription-benchmarking
+spec:
+  entrypoint: workflow-entry
+  templates:
+  - name: operation-one
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-one")
+  - name: operation-two
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-two")
+  - name: operation-three
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-three")
+  - name: operation-four
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-four")
+  - name: operation-five
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-five")
+  - name: operation-six
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-six")
+  - name: operation-seven
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-seven")
+  - name: operation-eight
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-eight")
+  - name: operation-nine
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-nine")
+  - name: operation-ten
+    script:
+      image: python:3.10
+      command: ["python"]
+      source: |
+        print("This is operation-ten")
+  - name: workflow-entry
+    dag:
+      tasks:
+      - name: operation-01
+        template: operation-one
+      - name: operation-02
+        template: operation-two
```

Each task needs to define which task it depends on to run one after the other.
```suggestion
        template: operation-two
        dependencies: [operation-01]
```